core/src/test/java/org/mule/util/scan/Grape.java

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.util.scan;
-
-import org.mule.tck.testmodels.fruit.Fruit;
-
-/**
- * TODO
- */
-public class Grape implements Fruit
```

Comment:
this class shouldn't be removed just beacuse we're removing ASM... This class doesn't even depend on it